eng/common/scripts/logging.ps1

```diff
@@ -31,10 +31,10 @@ function LogDebug
 {
     if ($isDevOpsRun) 
     {
-        Write-Host "##vso[task.LogIssue type=debug;]$args"
+        Write-Host "[debug]$args"
     }
     else 
     {
-        Write-Debug "$args"
+        Write-Debug "$args" -Debug
```

Comment:
Why do you want to always pass `-Debug` here? Don't you want to only have this write if someone passes `-Debug` as a common parameter? https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-7#debug

Reply:
I guess I can remove that. But it very helpful when running the tools locally.

Reply:
It is like Write-Verbose in that you don't want them to be displayed by default and only if you pass in the extra option should they be displayed.